src/org/thoughtcrime/securesms/ReactionsHandler.java

```diff
@@ -0,0 +1,151 @@
+package org.thoughtcrime.securesms;
+
+
+import android.content.Context;
+import android.database.Cursor;
+import android.os.Parcel;
+
+import org.thoughtcrime.securesms.database.Address;
+import org.thoughtcrime.securesms.database.DatabaseFactory;
+import org.thoughtcrime.securesms.database.MessageReactionDatabase;
+import org.thoughtcrime.securesms.database.model.MessageRecord;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ReactionsHandler {
+    private MessageReactionDatabase reactionsDb;
+
+    public ReactionsHandler(Context context) {
+        this.reactionsDb = DatabaseFactory.getMessageReactionDatabase(context);
+    }
+
+    /**
+     * NOT TEST
```

Comment:
should this be `NOT TESTED`?
also, I'm not sure the relevance of saying it is not tested

Reply:
yes, for this method and the one below it, remove the words "not tested" and perhaps make the comments a bit more concise.